policy_set.go

```diff
@@ -61,6 +64,8 @@ type PolicySet struct {
 	Name           string    `jsonapi:"attr,name"`
 	Description    string    `jsonapi:"attr,description"`
 	Global         bool      `jsonapi:"attr,global"`
+	PoliciesPath   string    `jsonapi:"attr,policies-path,omitempty"`
+	VCSRepo        *VCSRepo  `jsonapi:"attr,vcs-repo,omitempty"`
```

Comment:
I try to keep a (sort of) alphabetical order here, so I would prefer to reorder the new fields just a little:

```
	ID             string    `jsonapi:"primary,policy-sets"`
	Name           string    `jsonapi:"attr,name"`
	Description    string    `jsonapi:"attr,description"`
	Global         bool      `jsonapi:"attr,global"`
	PoliciesPath   string    `jsonapi:"attr,policies-path,omitempty"`
	PolicyCount    int       `jsonapi:"attr,policy-count"`
	VCSRepo        *VCSRepo  `jsonapi:"attr,vcs-repo,omitempty"`
	WorkspaceCount int       `jsonapi:"attr,workspace-count"`
	CreatedAt      time.Time `jsonapi:"attr,created-at,iso8601"`
	UpdatedAt      time.Time `jsonapi:"attr,updated-at,iso8601"`
```

Also the `omitempty` tags should be removed from the new fields in this struct.

Reply:
In terms of consistency `PolicyPath` would probably be a better name, but I guess that ship has sailed already :wink:

Reply:
This has been updated now!